src/test/java/org/broadinstitute/hellbender/utils/collections/IntervalsSkipListOneContigUnitTest.java

```diff
@@ -125,4 +135,56 @@ public void testManyIntervals() throws Exception {
                 +" indexed took "+v2.elapsed(TimeUnit.MILLISECONDS)+" ms, plus "+indexing.elapsed(TimeUnit.MILLISECONDS)+" for sorting&indexing.");
     }
 
+    @Test
+    public void testLotsOfTinyIntervals() throws Exception {
+        List<Locatable> input = new ArrayList<>();
+        int n = 1000000;
+        for (int i = 0; i < n; i++) {
+            input.add(new SimpleInterval("1", 3*i+1, 3*i+2)); //1:1-2, 1:4-5, 1:7-8
+        }
+        final IntervalsSkipListOneContig<Locatable> skipList = new IntervalsSkipListOneContig<>(input);
+        final List<Locatable> overlapping = skipList.getOverlapping(new SimpleInterval("1", 1, 3 * n + 2));
+        Assert.assertEquals(input, overlapping);
+    }
+
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testNullCtorArg() throws Exception {
+        new IntervalsSkipListOneContig<>(null);
+    }
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testNullArg() throws Exception {
+        List<Locatable> input = Arrays.asList(
+                new SimpleInterval("1",10,100)
+        );
+        final IntervalsSkipListOneContig<Locatable> l = new IntervalsSkipListOneContig<>(input);
+        l.getOverlapping(null);
```

Comment:
[FindBugs] INFO: NP: Null passed for nonnull parameter of IntervalsSkipListOneContig.getOverlapping(SimpleInterval) in org.broadinstitute.hellbender.utils.collections.IntervalsSkipListOneContigUnitTest.testNullArg()